tests/huggingface/sagemaker_dlc_test.py

```diff
@@ -75,7 +75,7 @@ def run_test(args):
     pytest.param("bigscience/bloom-560m", None, "ml.g5.12xlarge", marks=pytest.mark.gpu),
     pytest.param("EleutherAI/gpt-neox-20b", None, "ml.g5.12xlarge", marks=pytest.mark.gpu),
     pytest.param("google/flan-t5-xxl", None, "ml.g5.12xlarge", marks=pytest.mark.gpu),
-    pytest.param("HuggingFaceH4/zephyr-7b-beta", None, "ml.inf2.48xlarge", marks=pytest.mark.inf2),
+    pytest.param("NousResearch/Llama-2-7b-chat-hf", None, "ml.inf2.8xlarge", marks=pytest.mark.inf2),
```

Comment:
What is the change for?

Reply:
I am desperately trying to solve the CI issues, still blindly because I don't have access to the logs.
Feel free to add more models or fix the issues if you are able to figure out what went wrong (it used to work yesterday).

Reply:
I get ` sagemaker_dlc_test.TimeoutError: Test timed out` error

Reply:
This is the error you get eventually, but if you check the logs **interactively before** the time-out you can get the real error that blocks the deployment. This is why this should be addressed by someone with access to the logs: I can't continue pushing my changes blindly.

Reply:
I had made a mistake while changing model (dtype for Llama is fp16). Now the CI is green.

Reply:
We are prioritizing forwarding CW logs to GitHub, to make debugging easier